app/Models/Concerns/Filterable.php

```diff
@@ -161,12 +161,19 @@ public function scopeFilter(Builder $query, $params)
                             })
                             ->where('att.name', $filterValue);
 
-
                         $attributeValue = Arr::get($params, 'attribute_value');
-                        if ($attributeValue !== '' && $attributeValue !== null) {
+                        if ($attributeValue !== '' && $attributeValue !== null && $attributeValue !=='!!') {
                             $query
                             ->where('att.value', $attributeValue);
                         }
+
+                        if ($attributeValue == '!!') {
```

Comment:
Please use "===", this is for PHP to confirm the content and also the variable type (in this case a string) :)